testdata/configtest.json

```diff
@@ -257,11 +257,9 @@
      "delimiter": "-"
```

Comment:
can we revert all these changes, as this should be the old config? If this is erroring out on tests then we need to investigate.

Reply:
Revert all changes to this file PLEASE.